test/emails/nl/1.txt

```diff
@@ -0,0 +1,12 @@
+Hi Person,
+
+Message content goes here.
+
+Good to hear from you.
+
+This is still message content.
+
+Op do 22 mrt. 2019 om 13:58 schreef Person (Their Role) <
+conversation+2556@email.example.com>:
+>
```

Comment:
^Added this to the test emails to give us an example of a multi-line quote header.